packages/hardhat-core/src/internal/hardhat-network/stack-traces/error-inferrer.ts

```diff
@@ -460,6 +483,10 @@ export class ErrorInferrer {
 
       return this._fixInitialModifier(trace, inferredStacktrace);
     }
+
+    log(
+      "Last instruction is a revert or an invalid opcode, but couldn't infer anything"
```

Comment:
it's your old frenemy, passive voice.

```suggestion
      "Last instruction is a revert or an invalid opcode, but nothing could be inferred"
```